CHANGES.md

```diff
@@ -24,6 +24,99 @@ this file to understand what changed.
+    - Add feature metadata to IDataset and JointDataset by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/1614
+    - Remove console.log() statement from CohortEditor.tsx by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/1628
+    - update raiwidgets and responsibleai to erroranalysis 0.3.8 by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/1629
+    - add new interpret-vision package to npm publish workflows in build and release gates by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/1615
+    - [Accessibility] fix several accessibility bugs by @RubyZ10 in https://github.com/microsoft/responsible-ai-toolbox/pull/1632
+    - add UI tests for RAI text dashboard by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/1636
+    - Fix typo for indices by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/1640
+    - Add extra space between Identity feature and the actual feature name by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/1641
+    - Use JointDataset API in ProcessPreBuiltCohort.ts by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/1635
+    - add wrapping for long text documents to text highlighting component in RAI dashboard for text by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/1643
+    - add tests to RAI text dashboard importances part 2 by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/1642
+    - add radio button tests for RAI text dashboard by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/1648
+    - Localized file check-in by OneLocBuild Task: Build definition ID 21368: Build ID 69976515 by @tongyu-microsoft in https://github.com/microsoft/responsible-ai-toolbox/pull/1653
+    - Add unit test for calculateLineData by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/1654
+  - ## RAIInsights
+    - Refactor categorical statistics computation in separate module by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/1600
+    - responsibleai: Add FeatureMetadata class and related tests by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/1604
+    - Add categorical statistics computation in RAIInsights by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/1598
+    - Add cohort filtering capability in RAIInsights class by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/1618
+    - Inject FeatureMetadata into RAIInsights by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/1616
+    - responsibleai: Remove old manager namespaces by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/1564
+    - Add test for uncovered condition in responsibleai_dashboard_input.py by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/1638
```

Comment:
This should be in responsibleai dashboard section?

Reply:
done!